### ISSUE TITLE
(Feature request) Search all shared notes

### ISSUE THREAD

**Issue by passionate2023** [zadam/trilium#3346]
### Describe feature

I want to share all the notes via shared link

#1 - Link1 - can only view and cannot able to edit/delete
#2 - Link2 - can able to view, edit and delete
#3 - Link3 - The shared user should be able to view all the notes shared and search "all the shared notes" and not individual search on each note. He should be able to do all the operation like a normal user except the edit/delete is not allowed

Basically sharing info with friends but they are not supposed to delete but read/view/print but not edit/delete

**Comment by passionate2023** [comment]
#1 - exist but #2 and #3 - doesn't exist. Is my understanding is correct?

**Comment by sigaloid** [comment]
2 is just the same as having the regular login. It doesn't exist in the context of sharing notes. Outside of that, there's no such thing as multi users either.

**Comment by passionate2023** [comment]
Does sharing option -1 - gives all the remaining features as it is to the end users on view except edit/delete???

**Comment by sigaloid** [comment]
Sharing option just creates a publicly viewable HTML endpoint. It doesn't require a login and doesn't look like a Trilium installation but rather a regular website.

**Comment by passionate2023** [comment]
Need search, highlight and advanced search for the whole notebook which has been shared to the end user.

**Comment by zadam** [comment]
Tracked in https://github.com/zadam/trilium/issues/3430
